[2024-10-01T06:41:19.996+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-01T06:41:20.019+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip scheduled__2024-09-30T00:00:00+00:00 [queued]>
[2024-10-01T06:41:20.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip scheduled__2024-09-30T00:00:00+00:00 [queued]>
[2024-10-01T06:41:20.028+0000] {taskinstance.py:2306} INFO - Starting attempt 5 of 5
[2024-10-01T06:41:20.042+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_staging_layer.dbt_run_green_trip> on 2024-09-30 00:00:00+00:00
[2024-10-01T06:41:20.049+0000] {standard_task_runner.py:63} INFO - Started process 190 to run task
[2024-10-01T06:41:20.052+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Taxi_trip_pipeline', 'dbt_staging_layer.dbt_run_green_trip', 'scheduled__2024-09-30T00:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/taxi_trip.py', '--cfg-path', '/tmp/tmpa210tk8p']
[2024-10-01T06:41:20.054+0000] {standard_task_runner.py:91} INFO - Job 139: Subtask dbt_staging_layer.dbt_run_green_trip
[2024-10-01T06:41:20.101+0000] {task_command.py:426} INFO - Running <TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_green_trip scheduled__2024-09-30T00:00:00+00:00 [running]> on host fcb0f9e87e2b
[2024-10-01T06:41:20.184+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='Taxi_trip_pipeline' AIRFLOW_CTX_TASK_ID='dbt_staging_layer.dbt_run_green_trip' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-30T00:00:00+00:00'
[2024-10-01T06:41:20.185+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-01T06:41:20.197+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-01T06:41:20.198+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --profiles-dir /opt/***/DBT_Project  --project-dir /opt/***/dbt_trip --models green_trip']
[2024-10-01T06:41:20.212+0000] {subprocess.py:86} INFO - Output:
[2024-10-01T06:41:22.787+0000] {subprocess.py:93} INFO - [0m06:41:22  Running with dbt=1.8.0
[2024-10-01T06:41:23.102+0000] {subprocess.py:93} INFO - [0m06:41:23  Registered adapter: postgres=1.8.0
[2024-10-01T06:41:23.411+0000] {subprocess.py:93} INFO - [0m06:41:23  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-10-01T06:41:23.412+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2024-10-01T06:41:23.412+0000] {subprocess.py:93} INFO - - models.fact_trip.staging
[2024-10-01T06:41:23.412+0000] {subprocess.py:93} INFO - - models.fact_trip.Dimensions
[2024-10-01T06:41:23.640+0000] {subprocess.py:93} INFO - [0m06:41:23  Found 4 models, 6 data tests, 3 sources, 566 macros
[2024-10-01T06:41:23.642+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.777+0000] {subprocess.py:93} INFO - [0m06:41:23  Concurrency: 3 threads (target='dev')
[2024-10-01T06:41:23.778+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.790+0000] {subprocess.py:93} INFO - [0m06:41:23  1 of 1 START sql view model analysis.green_trip ................................ [RUN]
[2024-10-01T06:41:23.835+0000] {subprocess.py:93} INFO - [0m06:41:23  1 of 1 ERROR creating sql view model analysis.green_trip ....................... [[31mERROR[0m in 0.04s]
[2024-10-01T06:41:23.848+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.848+0000] {subprocess.py:93} INFO - [0m06:41:23  Finished running 1 view model in 0 hours 0 minutes and 0.20 seconds (0.20s).
[2024-10-01T06:41:23.901+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.902+0000] {subprocess.py:93} INFO - [0m06:41:23  [31mCompleted with 1 error and 0 warnings:[0m
[2024-10-01T06:41:23.903+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.904+0000] {subprocess.py:93} INFO - [0m06:41:23    Compilation Error in model green_trip (models/Staging/green_trip.sql)
[2024-10-01T06:41:23.905+0000] {subprocess.py:93} INFO -   'tripdata' is undefined. This can happen when calling a macro that does not exist. Check for typos and/or install package dependencies with "dbt deps".
[2024-10-01T06:41:23.905+0000] {subprocess.py:93} INFO - [0m06:41:23
[2024-10-01T06:41:23.906+0000] {subprocess.py:93} INFO - [0m06:41:23  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
[2024-10-01T06:41:24.979+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-10-01T06:41:24.979+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-01T06:41:24.994+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-10-01T06:41:24.997+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Taxi_trip_pipeline, task_id=dbt_staging_layer.dbt_run_green_trip, run_id=scheduled__2024-09-30T00:00:00+00:00, execution_date=20240930T000000, start_date=20241001T064120, end_date=20241001T064124
[2024-10-01T06:41:25.013+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 139 for task dbt_staging_layer.dbt_run_green_trip (Bash command failed. The command returned a non-zero exit code 1.; 190)
[2024-10-01T06:41:25.037+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-01T06:41:25.061+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-01T06:41:25.063+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
